### PR TITLE
docs: improve the desc of configuration items

### PR DESCRIPTION
**Pull request by tigercl**

### CODE REVIEW

File: desc.en.hocon [hunk 1]
```diff
@@ -1998,7 +1998,7 @@
       "label": "Connect Timeout"
     },
     "enable_pipelining": {
-      "desc": "A positive integer. Whether to send HTTP requests continuously, when set to 1, it means that after each HTTP request is sent, you need to wait for the server to return and then continue to send the next request.",
+      "desc": "The maximum number of HTTP requests that can be sent before an HTTP response is received. <br/>Setting this to 1 is equivalent to turning off HTTP Pipelining, and the EMQX must receive a response to the previous HTTP request before it can send the next HTTP request.",
```
Comment (zmstone) [inline desc.en.hocon:2001]:
```suggestion
      "desc": "The maximum number of HTTP requests that can be sent before an HTTP response is received. <br/>Setting this to 1 is equivalent to turning off HTTP Pipelining, and EMQX must receive a response to the previous HTTP request before sending the next HTTP request.",
```

**Comment by zmstone** [comment]
make the changes to en version descriptions here: https://github.com/emqx/emqx/tree/master/rel/i18n

File: desc.en.hocon [hunk 2]
```diff
@@ -4353,7 +4353,7 @@
       "label": "RPC Socket Sndbuf"
     },
     "rpc_ssl_server_port": {
-      "desc": "Listening port used by RPC local service.<br/>\nNote that this config only takes effect when rpc.port_discovery is set to manual\nand <code>driver</code> is set to <code>ssl</code>.",
+      "desc": "Listening port used by RPC local service.<br/>Note that this config only takes effect when `rpc.port_discovery` is set to manual and `rpc.driver` is set to `ssl`.",
```
Comment (zmstone) [inline desc.en.hocon:4356]: this has been changed in upstream: https://github.com/emqx/emqx/pull/13434
please consider sending a new commit to my PR instead.

**Comment by tigercl** [comment]
English changes moved to this PR: https://github.com/emqx/emqx/pull/13445

**Comment by Meggielqk** [comment]
I will add these updates to this pr: https://github.com/emqx/emqx-i18n/pull/22.

**Comment by zmstone** [comment]
will be merged as a part of #22

**Pull request closed without merging**